manifests/w/WacomTechnology,Corp/WacomTabletDriver/6.4.4-3/WacomTechnology,Corp.WacomTabletDriver.installer.yaml

```
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.5.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: WacomTechnology,Corp.WacomTabletDriver
+PackageVersion: 6.4.4-3
+InstallerType: exe
+Installers:
+- InstallerUrl: http://www.wacom.com/services/wacom/get-download-url.aspx?plat=win&dver=6.3.20-2&dt=drivers&redirect=true

```

```suggestion
- InstallerUrl: http://www.wacom.com/services/wacom/get-download-url.aspx?plat=win&dver=6.4.4-3&dt=drivers&redirect=true
```
While the URL with `6.3.20-2` retrieves the `6.4.4-3` version, would it be better to have the version in the URL?